src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlDateTimeMemberTranslator.cs

```diff
@@ -84,7 +93,8 @@ SqlFunctionExpression Now()
                     Array.Empty<SqlExpression>(),
                     nullable: false,
                     argumentsPropagateNullability: TrueArrays[0],
-                    returnType);
+                    returnType,
+                    _typeMappingSource.FindMapping(typeof(DateTime)));
```

Comment:
The PG now() function returns a `timestamp with time zone`, whereas DateTime maps to `timestamp without time zone`. Rather than looking up the mapping by CLR type (DateTime), you can look it up by the string store type (i.e. `FindMapping("timestamp with time zone")`).

However, it would be better to do this once in the constructor rather than looking up every time.

Reply:
OK, I set a variable in the constructor :
```csharp
_timestampTzTypeMapping = typeMappingSource.FindMapping("timestamp with time zone")!;
```
then I pass it to the function.